src/software/pipeline/main_cameraInit.cpp

```diff
@@ -476,17 +478,29 @@ int aliceVision_main(int argc, char **argv)
                 #pragma omp critical
                 viewsWithDCPMetadata++;
             }
-            else if (allColorProfilesFound)
+            else 
             {
-                // there is a missing color profile for one image or more
-                boost::atomic_ref<char>{allColorProfilesFound} = 0;
+                dcpError = true;
+                if (allColorProfilesFound)
+                {
+                    // there is a missing color profile for one image or more
+                    boost::atomic_ref<char>{allColorProfilesFound} = 0;
+                }
             }
         }
     }
 
     // Store the color interpretation mode choosed for raw images in metadata,
     // so all future loads of this image will be interpreted in the same way.
-    view.addMetadata("AliceVision:rawColorInterpretation", image::ERawColorInterpretation_enumToString(rawColorInterpretation));
+    if (!dcpError)
+    {
+        view.addMetadata("AliceVision:rawColorInterpretation", image::ERawColorInterpretation_enumToString(rawColorInterpretation));
+    }
+    else
+    {
+        view.addMetadata("AliceVision:rawColorInterpretation", image::ERawColorInterpretation_enumToString(image::ERawColorInterpretation::LibRawWhiteBalancing));
+        ALICEVISION_LOG_WARNING("DCP Profile not found. Use LibRawWhiteBalancing option for raw color processing.");
```

Comment:
It would be good to add the path of the image and only print if there a database specified.
```
        if (!dcpDatabase.empty())
            ALICEVISION_LOG_WARNING("DCP Profile not found for image: " << imagePath<< ". Use LibRawWhiteBalancing option for raw color processing.");
```

Reply:
Done!